javascripts/govuk/anchor-buttons.js

```diff
@@ -0,0 +1,55 @@
+(function(global, settings) {
+  "use strict"; 
+       
+  var $ = global.jQuery;
+  var GOVUK = global.GOVUK || {};  
+
+  GOVUK.anchorButtons = {
+    
+    // default configuration that can be overridden by passing object as second parameter to module
+    config: $.extend({
+      // setting to true will disable the init function upon execution
+      disableInit: false,
+      // the target element(s) to attach the shim event to
+      selector: 'a[role="button"]',
```

Comment:
I wasn't sure about that one. Do we always want space to trigger a click on something with ```role="button"``` ? @LJWatson @aduggin to advise perhaps?